### DISCUSSION TITLE
昨天shinchiro出了个x86_64-v3版本的mpv

### DISCUSSION THREAD

**Discussion by yeezylife** [hooke007/mpv_PlayKit#166] [Q&A 问答] [answered]
说是要支持avx和avx2指令集才能用这个v3版本，我的CPU倒是支持avx2啦。
但我想知道：
1.这个需要avx2指令集支持的v3版本叼在哪里？（会让mpv性能更强么）
2.之前的那些着色器啊 脚本什么的都兼容么？

**Comment by hooke007** [comment, marked as answer]
编译层的优化
https://github.com/nyfair/mpv-win64/issues/16#issuecomment-1221756303

终端用户除了程序运行兼容性不需要关心它

**Reply by yeezylife** [reply]
更新了新版osc_lazy.lua， 确实不需要osc=no了 

但是刚才我屏幕自动关闭了（此时全屏模式下暂停着一个视频，with `gpu-api=d3d11` `fs=yes`
`d3d11-exclusive-fs=yes` ）， 我移动鼠标点亮屏幕，在全屏模式下把鼠标移动到右上角想点击最小化-按钮，发现点不动（此时其他按钮比如关闭x也点不动）。

用快捷键esc退出mpv后，再次打开其他任何视频 osc界面都显示不出来了（鼠标移到顶部或者底部没有osc出来）。

 我删除portable_config文件夹之后重开恢复了。  好奇怪的bug 我用之前版本的osc_lazy.lua（黑屏，再移动鼠标亮屏）从未出现过这种情况，等会我吃饭的时候再试试能不能复现（开个视频暂停让它自动黑屏）

我需要为这个单独开个issue么

**Reply by hooke007** [reply]
没有稳定的复现方式只能当玄学。
而且你的环境太杂了，单脚本无配置测试看有没有问题，至于是不是其他的冲突得一点点增补测了

**Reply by yeezylife** [reply]
黑屏导致是我的误会了，我现在看着看着 已经无法唤出osc了（播放了大概8分钟），我退回之前的版本好了。。。

**Reply by hooke007** [reply]
这个新加的功能本身是上保险，防止osc被手动开启/别的脚本调用原osc。除此之外也没别的变动。

不需要回退，你把设置里的osc=no加回去不就完了（我也不确定osc在启动后再禁用的方式退的干不干净，没测稳定前也不会发新版

**Reply by yeezylife** [reply]
OK 那我试试新版+ `osc=no` 如果还有问题再回复你

**Reply by hooke007** [reply]
我知道问题了，我之前给别的脚本推pr https://github.com/tomasklaen/uosc/pull/128 的时候是没问题的因为它的界面自带渲染延迟，但是原osc和这个基于osc修改的版本没有，所以会概率性触发渲染冲突。所以恢复 https://github.com/hooke007/MPV_lazy/commit/f9ef63cd32da9852779c2b3b0bb0d05304b8196d mpv.conf部分的提交

**Reply by yeezylife** [reply]
总之我的测试就是开 osc=no 就行，   不开就出问题
